database/seeds/SettingsTableSeeder.php

```diff
@@ -95,5 +95,12 @@ public function run()
       'value'     => 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Eaque facilis, explicabo quidem ea dolore fugit.',
       'type'      => 'textarea',
     ]);
+
+      Setting::create([
+      'category'  => 'general',
```

Comment:
@angaither I need to go through and clean up the rest of the db seeds, they are all over the place, but is how this one is now what we want across the board?